components/Notifications/SubscribedAuthors.js

```diff
@@ -153,3 +176,20 @@ const SubscribedAuthors = ({
 }
 
 export default compose(withT, graphql(myUserSubscriptions))(SubscribedAuthors)
+
+function InitializeSubscribedAuthorIds(
```

Comment:
The function name I'd start with lowercase. To be more consistent with the rest of the codebase, I'd also bring it inside the component and use `setInitialySubscribedAuthorIds`directly. That way, the `export default`also remains the last line of the file, which is also consistent with our codebase.